be/src/exec/olap_scan_node.cpp

```diff
@@ -1566,7 +1566,7 @@ void OlapScanNode::scanner_thread(OlapScanner* scanner) {
     auto& scanner_filter_apply_marks = *scanner->mutable_runtime_filter_marks();
     DCHECK(scanner_filter_apply_marks.size() == _runtime_filter_descs.size());
     for (size_t i = 0; i < scanner_filter_apply_marks.size(); i++) {
-        if (!scanner_filter_apply_marks[i] && !_runtime_filter_ctxs[i].apply_mark) {
```

Comment:
should keep the  `!_runtime_filter_ctxs[i].apply_mark`.
because if `runtime filter` come before 1s, the `apply_mark` is `true`, we do not need dispose the `runtime filter` already push down

Reply:
done